[2025-06-29T18:12:02.292+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: films_dag_orchestaror.dbt_task scheduled__2025-06-29T18:11:00+00:00 [queued]>
[2025-06-29T18:12:02.300+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: films_dag_orchestaror.dbt_task scheduled__2025-06-29T18:11:00+00:00 [queued]>
[2025-06-29T18:12:02.300+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-29T18:12:02.312+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): dbt_task> on 2025-06-29 18:11:00+00:00
[2025-06-29T18:12:02.316+0000] {standard_task_runner.py:60} INFO - Started process 667 to run task
[2025-06-29T18:12:02.321+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'films_dag_orchestaror', 'dbt_task', 'scheduled__2025-06-29T18:11:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/orchestrator.py', '--cfg-path', '/tmp/tmp1xkb5w3n']
[2025-06-29T18:12:02.323+0000] {standard_task_runner.py:88} INFO - Job 59: Subtask dbt_task
[2025-06-29T18:12:02.375+0000] {task_command.py:423} INFO - Running <TaskInstance: films_dag_orchestaror.dbt_task scheduled__2025-06-29T18:11:00+00:00 [running]> on host 9909ba106c7b
[2025-06-29T18:12:02.464+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='films_dag_orchestaror' AIRFLOW_CTX_TASK_ID='dbt_task' AIRFLOW_CTX_EXECUTION_DATE='2025-06-29T18:11:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-29T18:11:00+00:00'
[2025-06-29T18:12:02.499+0000] {docker.py:359} INFO - Starting docker container from image ghcr.io/dbt-labs/dbt-postgres:1.9.latest
[2025-06-29T18:12:02.507+0000] {docker.py:367} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2025-06-29T18:12:04.335+0000] {docker.py:429} INFO - [0m18:12:04  Running with dbt=1.9.0
[2025-06-29T18:12:04.635+0000] {docker.py:429} INFO - [0m18:12:04  Registered adapter: postgres=1.9.0
[2025-06-29T18:12:04.834+0000] {docker.py:429} INFO - [0m18:12:04  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_dbt_project.example
[2025-06-29T18:12:04.992+0000] {docker.py:429} INFO - [0m18:12:04  Found 3 models, 3 sources, 431 macros
[2025-06-29T18:12:04.995+0000] {docker.py:429} INFO - [0m18:12:04
[2025-06-29T18:12:04.996+0000] {docker.py:429} INFO - [0m18:12:04  Concurrency: 4 threads (target='dev')
[2025-06-29T18:12:04.997+0000] {docker.py:429} INFO - [0m18:12:04
[2025-06-29T18:12:05.106+0000] {docker.py:429} INFO - [0m18:12:05  1 of 3 START sql table model dev.film_rating ................................... [RUN]
[2025-06-29T18:12:05.107+0000] {docker.py:429} INFO - [0m18:12:05  2 of 3 START sql table model dev.staging ....................................... [RUN]
[2025-06-29T18:12:05.273+0000] {docker.py:429} INFO - [0m18:12:05  2 of 3 OK created sql table model dev.staging .................................. [[32mSELECT 580[0m in 0.16s]
[2025-06-29T18:12:05.275+0000] {docker.py:429} INFO - [0m18:12:05  1 of 3 OK created sql table model dev.film_rating .............................. [[32mSELECT 580[0m in 0.17s]
[2025-06-29T18:12:05.279+0000] {docker.py:429} INFO - [0m18:12:05  3 of 3 START sql table model dev.film_actors ................................... [RUN]
[2025-06-29T18:12:06.175+0000] {docker.py:429} INFO - [0m18:12:06  3 of 3 OK created sql table model dev.film_actors .............................. [[32mSELECT 487780[0m in 0.89s]
[2025-06-29T18:12:06.194+0000] {docker.py:429} INFO - [0m18:12:06
[2025-06-29T18:12:06.194+0000] {docker.py:429} INFO - [0m18:12:06  Finished running 3 table models in 0 hours 0 minutes and 1.20 seconds (1.20s).
[2025-06-29T18:12:06.238+0000] {docker.py:429} INFO - [0m18:12:06
[2025-06-29T18:12:06.239+0000] {docker.py:429} INFO - [0m18:12:06  [32mCompleted successfully[0m
[2025-06-29T18:12:06.240+0000] {docker.py:429} INFO - [0m18:12:06
[2025-06-29T18:12:06.241+0000] {docker.py:429} INFO - [0m18:12:06  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2025-06-29T18:12:07.325+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=films_dag_orchestaror, task_id=dbt_task, execution_date=20250629T181100, start_date=20250629T181202, end_date=20250629T181207
[2025-06-29T18:12:07.372+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-29T18:12:07.395+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
